sql/moz-fx-data-shared-prod/firefox_accounts/docker_fxa_admin_server_sanitized/view.sql

```diff
@@ -38,7 +38,8 @@ SELECT
         jsonPayload.fields.search_type
       ) AS fields
     ) AS jsonPayload
-  )
+  ),
+  NULL AS errorGroups
```

Comment:
Not sure this will work since error groups in moz-fx-data-shared-prod.firefox_accounts_derived.docker_fxa_admin_server_sanitized_v2 is an array with a struct

Reply:
might need something like [struct(null as id)] as errorGroups ...

Reply:
It does, check `mozdata.tmp.akomar_docker_fxa_admin_server_sanitized`.